parser_test.go

```diff
@@ -17,13 +17,21 @@ func runParserWith(line string) *Parser {
 }
 
 func TestParserWithMatchingInputGo16(t *testing.T) {
-	line := "gc #88 @3.243s 9%: 0.040+16+1.0+5.9+0.34 ms clock, 0.16+16+0+18/5.7/11+1.3 ms cpu, 32->33->19 MB, 33 MB goal, 4 P"
+	line := "gc 763 @77536.239s 1%: 0.11+2192+0.75 ms clock, 0.92+9269/4379/3243+6.0 ms cpu, 6370->6390->3298 MB, 6533 MB goal, 8 P"
```

Comment:
this *is* the test for go 1.6. the original test for 1.6 was incorrectly using the 1.5 format which had extra fields. it's only 1.5 that has the extra fields.